### DISCUSSION TITLE
What should I do if I support DirectX?

### DISCUSSION THREAD

**Discussion by kaiserinhack** [dotnet/maui#8559] [General]
I am bad at English
I Made Proguram
What make to maui CreateDeviceAndSwapChain DirectRender what？

**Comment by Xyncgas** [comment]
Hello~ Do you know what I am saying? I like you very much ahh thank you for being our membership.
I can tell you about more DirectX, it's a platform supported by the microsoft windows.
You can use DirectX normally as you would on other types of native .NET projects if you are running windows because DirectX only supports windows I hope you like what I am saying thanks

https://stackoverflow.com/questions/28867607/using-d3d11createdeviceandswapchain#28868192
